core/dhcpserver/server.go

```diff
@@ -144,24 +145,15 @@ func (s *Server) serveDHCPv4(c net.PacketConn, payload []byte, addr net.Addr) er
 	// as per RFC2131
 	resp.UpdateOption(dhcpv4.OptServerIdentifier(cfg.IP))
 
-	switch msg.MessageType() {
```

Comment:
I've forgot why I removed this part XD. But now when I look at this again, It seems to be wrong. 
I will test it with on vm.

Reply:
@fooxlj07 Sure, let me know.